var/spack/repos/builtin/packages/exawind/package.py

```diff
@@ -6,35 +6,93 @@
 from spack.package import *
 
 
-class Exawind(CMakePackage):
+class Exawind(CMakePackage, CudaPackage, ROCmPackage):
     """Multi-application driver for Exawind project."""
 
     homepage = "https://github.com/Exawind/exawind-driver"
     git = "https://github.com/Exawind/exawind-driver.git"
 
-    maintainers("jrood-nrel", "psakievich")
+    maintainers = ["jrood-nrel"]
```

Comment:
Can you use the directive:
```suggestion
    maintainers("jrood-nrel")
```
?